at-chuongphu/excercise3.rb

```diff
@@ -0,0 +1,49 @@
+# ex1
+print 'Enter score: '
+score = gets.to_f
+if score < 6.0
+  puts 'F'
+elsif score >= 9.0
+  puts 'A'
+elsif score >= 6.0 && score < 7.0
+  puts 'D'
+elsif score >= 7.0 && score < 8.0
+  puts 'C'
+elsif score >= 8.0 && score < 9.0
+  puts 'B'
+else
+  puts 'invailid'
+end
+# ex2
+a = [1, 2, 3, 1]
+a.map! do |_number|
+  x = if numer.even?
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/f1fe9b514a37a04f5d31692cea30d1d82c4b1a3e/at-chuongphu/excercise3.rb#L20"></span>Lint/UselessAssignment: Useless assignment to variable - <code>x</code>. (https://github.com/rubocop-hq/ruby-style-guide#underscore-unused-vars)</td>
    </tr>
  </tbody>
</table>